### PR TITLE
Support for Sybase ASE and Informix 

### PR DESCRIPTION
**Pull request by anasY2**

### CODE REVIEW

File: src/main/java/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java
Comment (apavlo) [inline src/main/java/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java:1]: What changed in this file other than whitespace?

File: src/main/java/com/oltpbenchmark/benchmarks/ycsb/YCSBBenchmark.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ protected List<Worker<? extends BenchmarkModule>> makeWorkersImpl() {
             // LOADING FROM THE DATABASE IMPORTANT INFORMATION
             // LIST OF USERS
             Table t = this.getCatalog().getTable("USERTABLE");
-            String userCount = SQLUtil.getMaxColSQL(this.workConf.getDatabaseType(), t, "ycsb_key");
+            String userCount = SQLUtil.getMaxColSQL(this.workConf.getDatabaseType(), t, "YCSB_KEY");
```
Comment (apavlo) [inline src/main/java/com/oltpbenchmark/benchmarks/ycsb/YCSBBenchmark.java:80]: Is this change specific to Informix?

File: src/main/java/com/oltpbenchmark/types/DatabaseType.java [hunk 2]
```diff
@@ -35,7 +35,8 @@ public enum DatabaseType {
     DB2(true, false),
     H2(true, false),
     HSQLDB(false, false),
-    POSTGRES(false, false, true),
+    INFORMIX(false, false),
+    POSTGRES(false, false),
```
Comment (apavlo) [inline src/main/java/com/oltpbenchmark/types/DatabaseType.java:39]: Why did you change Postgres?

File: src/main/java/com/oltpbenchmark/types/DatabaseType.java [hunk 3]
```diff
@@ -47,8 +48,9 @@ public enum DatabaseType {
     SPANNER(false, true),
     SQLAZURE(true, true, true),
     SQLITE(true, false),
-    SQLSERVER(true, true, true),
-    TIMESTEN(true, false),
+    SQLSERVER(true, false),
```
Comment (apavlo) [inline src/main/java/com/oltpbenchmark/types/DatabaseType.java:51]: Why did you change SQLSERVER?

File: src/main/resources/benchmarks/tatp/ddl-sqlserver.sql
Comment (apavlo) [inline src/main/resources/benchmarks/tatp/ddl-sqlserver.sql:1]: Why did you change syntax case?

File: src/main/resources/benchmarks/tpcc/ddl-sqlserver.sql
Comment (apavlo) [inline src/main/resources/benchmarks/tpcc/ddl-sqlserver.sql:1]: Why did you change syntax case?

File: src/main/resources/benchmarks/voter/ddl-sqlserver.sql
Comment (apavlo) [inline src/main/resources/benchmarks/voter/ddl-sqlserver.sql:1]: Again, these changes seem unnecessary.

File: src/main/resources/benchmarks/ycsb/ddl-sqlserver.sql
Comment (apavlo) [inline src/main/resources/benchmarks/ycsb/ddl-sqlserver.sql:1]: Unnecessary case changes. Please revert.

File: src/main/resources/logging.properties
Comment (apavlo) [inline src/main/resources/logging.properties:1]: Please don't include this change. It's only for local dev debugging.

File: pom-arcion.xml
Comment (apavlo) [inline pom-arcion.xml:1]: What is this file?

**Review by apavlo — CHANGES_REQUESTED** [review]
@anasY2 Thank you for submitting this. Please see my comments about your commits.

Is there a way for us to add this to our CI tests with a Docker installation of Informix/Sybase? It would be nice to know that we can maintain compatibility with these systems over the years.

**Comment by anasY2** [comment]
@apavlo I will correct the things you have mentioned and will make a new PR.
Thankyou for suggestions :).

**Pull request closed without merging**